decidim-generators/lib/decidim/generators/app_templates/initializer.rb

```diff
@@ -455,8 +453,6 @@
     unless Rails.application.secrets.dig(:decidim, :initiatives, :creation_enabled) == "auto"
       config.creation_enabled = Rails.application.secrets.dig(:decidim, :initiatives, :creation_enabled).present?
     end
-    config.similarity_threshold = Rails.application.secrets.dig(:decidim, :initiatives, :similarity_threshold).presence || 0.25
```

Comment:
Even though we're still not dropping the support for this feature in `decidim-initiatives` yet, I plan to do it in a future PR before the v0.29, so let's leave these removal too.